pkg/user/mongo/user_test.go

```diff
@@ -0,0 +1,57 @@
+package mongo
+
+import (
+	"testing"
+	"time"
+
+	"github.com/gomods/athens/pkg/user"
+)
+
+var m *MongoUserStore
+
+func TestConnect(t *testing.T) {
+	setup(t)
+}
+
+func TestCreate(t *testing.T) {
+	m := setup(t)
+	defer m.s.Close()
+	g := &user.User{}
+	g.AccessToken = "accesstoken"
+	g.AccessTokenSecret = "accesstokensecret"
+	g.AvatarURL = "https://avatar.url"
+	g.Description = "description"
+	g.Email = "goth@goth.user"
+	g.ExpiresAt = time.Now()
+	g.FirstName = "Goth"
+	g.LastName = "User"
+	g.Location = "Buffalo, NY"
+	g.Name = "Goth User"
+	g.NickName = "guser"
+	g.Provider = "github"
+	g.RefreshToken = "refreshtoken"
+	g.UserID = "012345678"
```

Comment:
are all those field needed, don't know about github but microsoftonline does not provide them all

Reply:
technically no, but these are the base sets of fields from goth.